app/styles/globals/_colors.scss

```diff
@@ -19,61 +19,4 @@
   --titles-color: hsl(222, 50%, 14%);
   --section-bg-color: hsl(222, 50%, 14%);
   --tertiary-dark-mode: hsl(222, 52%, 5%);
-
-  $clr-primary-light-mode: hsl(0, 0%, 100%);
-  $clr-secondary-light-mode: hsl(31, 94%, 86%);
-  $clr-tertiary-light-mode: hsl(222, 50%, 14%);
-  $clr-text-light-mode: hsl(215, 19%, 35%);
-  $clr-elevated-elements-light-mode: hsl(210, 40%, 98%);
-  $clr-titles-light-mode: hsl(222, 50%, 14%);
-  $clr-tertiary-button-light-mode: hsl(0, 0%, 0%);
-
-  $clr-primary-dark-mode: #121d36;
-  $clr-secondary-dark-mode: hsl(222, 19%, 44%);
-  $clr-tertiary-dark-mode: hsl(222, 52%, 5%);
```

Comment:
@aleksandarmicev  Ace is this a mistake or on purpose? I see you have deleted all the colors from colors.scss

Reply:
It's not a mistake, I purpose in the code to exist separate file for color variables, and in color to be only primary colors

Reply:
why would we do that?

Reply:
Please undo that.
This ticket is only for updating the colors used in the Mission/Vision section.
We can discuss updating the Sass file structure. Maybe we can use the 7:1 method. But we should create a separate issue for that. 🙏 
Try to keep the PRs only about what was mentioned in the ticket.

Reply:
Ok, I'll revert these changes.

Reply:
Thank you